readthedocs/settings/base.py

```diff
@@ -326,6 +326,16 @@ def USE_PROMOS(self):  # noqa
             'schedule': crontab(minute=0, hour='*/3'),
             'options': {'queue': 'web'},
         },
+        'every-three-month-delete-old-search-queries': {
+            'task': 'readthedocs.search.tasks.delete_old_search_queries_from_db',
+            'schedule': crontab(
+                minute=0,
+                hour=0,
+                day_of_month=1,
+                month_of_year='*/4'
```

Comment:
We want to run this more frequently, but only delete the last 3 months of data. I imagine it running daily.